tools/ci/testlist/sim-02.dat

```diff
@@ -33,6 +33,16 @@
 # macOS matter compilation is not currently supported
 -Darwin,sim:matter
 
+# macOS doesn't support cxxabi.h
+-Darwin,sim:libcxxtest
+
+# macOS doesn't support pipe2()
+-Darwin,sim:lua
```

Comment:
but pipe2 should work in Darwin

Reply:
Something special about the Lua build I think, since the other calls to pipe2 seem OK?

Reply:
I have created the issue:
- https://github.com/apache/nuttx/issues/14773